server/src/items/classes/spell_scrolls/WardSpellScroll.js

```diff
@@ -0,0 +1,26 @@
+const SpellScroll = require("./SpellScroll");
```

Comment:
I should be ok to move the MagicEffect import to the top here `const MagicEffects = require("../../../gameplay/MagicEffects");`

so you don't need to bother doing
`const me = this;`
as it can be used directly from the outermost file scope as
`new MagicEffects.Ward(destroyable);`